src/flownet/config_parser/_config_parser_pred.py

```diff
@@ -1,26 +1,46 @@
 import os
 import pathlib
-from typing import Dict
+from typing import Dict, Optional
 
 import yaml
 import configsuite
 from configsuite import types, MetaKeys as MK, ConfigSuite
 
 from ._merge_configs import merge_configs
+from ._config_transformations import _to_upper
 
 
-def create_schema(_to_abs_path) -> Dict:
+def create_schema(config_folder: Optional[pathlib.Path] = None) -> Dict:
     """
     Returns a configsuite type schema, where configuration value types are defined, together
     with which values are optional and/or has default values.
 
     Args:
-        _to_abs_path: Use absolute path transformation
-
+        config_folder: Optional path to folder with config file
     Returns:
         Dictionary to be used as configsuite type schema
 
     """
+
+    @configsuite.transformation_msg("Convert input string to absolute path")
```

Comment:
Move this one also to `_config_transformations`?

Reply:
I wanted to. However, because this method uses the local variable config_folder I didn't get it to work. I spend about half an hour trying to pass an argument to a transformation function but didn't get it to work. I agree it would be nicer, but maybe other things have a higher priority right now.